admin_guide/manage_users.adoc

```diff
@@ -9,15 +9,21 @@
 
 toc::[]
 
+[[managing-users-adding-users]]
```

Comment:
Normally the topics start with an "Overview" section, intro'ing the topic. Would one make sense here?